server/service/src/document/raw_document.rs

```diff
@@ -90,9 +93,10 @@ mod document_id_test {
             comment: None,
             owner_name_id: None,
             context: None,
+            is_sync_update: false,
         };
         let document = raw.finalise().unwrap();
-        let expected_json_string = r#"{"author":"author","data":{"a":"avalue","b":0.3453333},"name":"name","parents":["p1"],"status":"Active","timestamp":"1970-01-01T00:00:01Z","type":"test"}"#;
+        let expected_json_string = r#"{"author":"author","data":{"a":"avalue","b":0.3453333},"is_sync_update":false,"name":"name","parents":["p1"],"status":"Active","timestamp":"1970-01-01T00:00:01Z","type":"test"}"#;
```

Comment:
This should be included here, the hash is for the raw data while `is_sync_update` is some internal state...